u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AndroidKeyStoreAttestation.java

```diff
@@ -135,11 +140,15 @@ private AndroidKeyStoreAttestation(Integer keymasterVersion, byte[] attestationC
    *       patchMonthYear  INTEGER,
    *   }
    */
-  public static AndroidKeyStoreAttestation Parse(X509Certificate cert)
-      throws CertificateParsingException {
-    // Extract the extension from the certificate
+  public static AndroidKeyStoreAttestation Parse(
+      X509Certificate[] certChain, X509Certificate[] caCerts) throws CertificateParsingException {
```

Comment:
Meta-comment, not for this CL: it seems weirdly asymmetrical to pass caCerts here, but have a different concept for trusted roots elsewhere. Could we try to have a consistent approach for what a trusted root?